src/com/dre/brewery/lore/BrewLore.java

```diff
@@ -173,8 +176,9 @@ public void updateAgeLore(boolean qualityColor) {
 		if (brew.isStripped()) return;
 		String prefix;
 		float age = brew.getAgeTime();
+		int quality = brew.getIngredients().getAgeQuality(brew.getCurrentRecipe(), age);
```

Comment:
Thanks for your contribution, my server is using this pr :), but also found a little problem. brew.getCurrentRecipe() may return null (if someone made Ruined Potion)